person.rb

```diff
@@ -0,0 +1,35 @@
+require './corrector'
+require './rental'
+
+class Person
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(9999)
+    @parent_permission = parent_permission
+    @age = age
+    @name = name
+    @corrector = Corrector.new
+    @rentals = []
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
```

Comment:
- [ ] with the previous comment I meant that the rental class should be initialized similarly to below:
```
 def add_rental(book, date)
    Rental.new(date, book, self)
  end
```